docs/usage.rst

```diff
@@ -172,21 +192,19 @@ We can remove files.
     interface.remove_file("/archives/folder.zip")
     interface.ls("/")
 
-Or even delete the whole repository
-with all its content.
+Finally,
+we close the connection to the backend.
 
 .. jupyter-execute::
 
-    audbackend.backend.FileSystem.delete("host", "repo")
-
+    backend.close()
 
-We can check if a repository exists
-by inspecting its root path.
```

Comment:
As we no longer can call exists, I think we also need a method/way to check if the repository exists.
At the moment `open()` seems to be the solution? If yes, we should maybe add it here instead of deleting the example.
On the other hand, `open()` feels not very intuitive to me to check if the repo exists.

Reply:
Yes, maybe such a function makes sense. It's a bit unfortunate that we called the function to check if a file exists `exists()` and not `exists_file()` as with all the other functions like `put_file()`, etc.

Reply:
Btw: it's a bit strange that it was working before. When I delete a repository I would expect that the folder is removed, too. But it seems that this is not the case since `exists('/')` was not raising an error when called after `delete()`.

Reply:
And apart from that it is also strange that is is possible to call `exists()` with `/` which is not the path to a file.

Reply:
`exists('/')` returned `False` for file-system as the folder did not exists. What is not deleted by `delete()` is the host (as for some backends we can not delete the host), whereas `create()` also cretaes the host folder for `FileSystem`.

Reply:
Yes and no. You are right that we state that `path` should be a file path, but we do also not discuss what should happen if you provide `/` as path in the docstring:

![image](https://github.com/audeering/audbackend/assets/173624/37e441e2-5295-4077-90ae-1ebc19091927)

Reply:
I think we still have two options here (that we should handle in a separate pull request):
* raise an error for `"/"` as `path` and extend the `Raises` section accordingly
* check for `"/"` if the repository exists and extend the docstring accordingly

Reply:
> raise an error for "/" as path and extend the Raises section accordingly

Not specifically `"/"`, but any path that ends on `"/"`. One solution would be to extend `utils.check_path()` accordingly and then extend the description of the first `ValueError`. This applies to all functions except `ls()`, which is the only function that accepts a sub-path.

> check for "/" if the repository exists and extend the docstring accordingly

I thought about that, too. But what I do not like about this solution is that usually you call `open()` first (at least on some backends) before you can call `exists()`. But then with `'/'` as argument you would call it **before** `open()` to check if you can call `open()`. That is slightly confusing.